js/components/conceptpicker/ConceptPicker.js

```diff
@@ -27,7 +27,13 @@ define(['jquery','knockout', 'text!./ConceptPickerTemplate.html', './InputTypes/
 		self.dtApi = ko.observable();
 
 		VocabularyProvider.getDomains().then(function (domains) {
-			self.DomainOptions(domains);
+			if (self.SelectedDomain != null) {
+				const domainList = domains.filter(d => d != self.SelectedDomain); // moving selected domain to top
+				self.DomainOptions([self.SelectedDomain].concat(domains));
```

Comment:
You mean `[self.SelectedDomain].concat(domainList)`? 

And in the first place, why do you want selected domain on top of the list? Popup already opens with selected domain.

Reply:
Yes, I shoudl ahve put domainList in the concat. Good catch, thank you!

I am doing this operation to handle the case where if the domain doesn't exist yet (as in the case of the domain 'Condition Status'):  if a default domain is specified, I still want it to appear in the list, even if it's not found from the vocabulary search.  (the specific case of the 'Condition Status' domain is more complicated than I want to go into here, but I'm trying to accommodate the random-feeling changes that are happening in the vocabulary).  Putting in the 'selected' domain on the top was just a nice side-effect.

I'll push up the fix to the right variable, but I hope the overall approach here makes sense.